jvb/src/main/kotlin/org/jitsi/videobridge/message/BridgeChannelMessage.kt

```diff
@@ -241,10 +259,12 @@ class ReceiverVideoConstraintMessage(val maxFrameHeight: Int) : BridgeChannelMes
 /**
  * A message sent from the bridge to a client, indicating that the dominant speaker in the conference changed.
  */
+@JsonInclude(JsonInclude.Include.NON_NULL)
 class DominantSpeakerMessage @JvmOverloads constructor(
-    var dominantSpeakerEndpoint: String,
+    dominantSpeakerEndpoint: String,
     val previousSpeakers: List<String>? = null
 ) : BridgeChannelMessage(TYPE) {
+    var dominantSpeakerEndpoint: String by observableWhenChanged(dominantSpeakerEndpoint) { -> resetJsonCache() }
```

Comment:
Apparently this is a consice way to diambiguate between `_ -> ...`  and `_, _, _ -> ...`